microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -1107,49 +1006,49 @@ See also:
 * [Application tool, to manage storage](../storage/application-tool.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 128.
-* This checkbox is present in Microsoft Edge Stable 125.
-
-
-<!-- ====================================================================== -->
-## Enable Speculative Loads Panel in Application panel
-<!-- checkbox has no (?) link pointing to this anchor wording -->
-
-_new as of Microsoft Edge 121_
-
-See the above section, showing the newer label for the checkbox.
+* This checkbox is present in Microsoft Edge Canary 129.
+* This checkbox is present in Microsoft Edge Stable 127.
 
 
 <!-- ====================================================================== -->
-## Enable Storage Buckets Tree in Application panel
-<!-- checkbox has no (?) link pointing to this anchor wording -->
+## Performance panel: Enable annotations
 
-_new as of Microsoft Edge 118_
+_new as of Microsoft Edge Canary 129_
 
-Whether to enable the Storage Buckets Tree in the **Application** tool.
-<!-- todo old: need text -->
+Enables annotations in the Performance tool.
+<!-- todo: need desc -->
 
 See also:
-* [Application tool, to manage storage](../storage/application-tool.md)
+* [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is not present in Microsoft Edge Canary 128.
-* This checkbox is present in Microsoft Edge Stable 125.
+* This checkbox is present in Microsoft Edge Canary 129.
+* This checkbox is not present in Microsoft Edge Stable 127.
 
 
 <!-- ====================================================================== -->
-## Enable the experimental, WIP sidebar for the Performance Panel.
+## Performance panel: Enable sidebar
 <!-- checkbox has no (?) link pointing to this anchor wording -->
-<!-- checkbox label has period -->
 
 _new as of Microsoft Edge 128_
 
-Enables the sidebar in the **Performance** tool.  The sidebar is an experimental work in progress (WIP).
+Enables the sidebar in the **Performance** tool.
+<!-- todo: details -->
```

Comment:
need details for **Performance panel: Enable sidebar** experiment checkbox - what content is in this sidebar?
@zoherghadyali @leahmsft @captainbrosset